lib/commands/new/wizard-helper.js

```diff
@@ -151,8 +151,7 @@ function _newWithCustomTemplate(templateUrl, doDebug, callback) {
         });
     } else {
         process.nextTick(() => {
-            callback(`The provided template url ${templateUrl} is not a supported type. \
-We currently only support ".git" url for user's custom template.`);
+            callback(`The provided template url ${templateUrl} is not a valid url.`);
```

Comment:
I tried with url that is not git-clone-able, and get the following experiences:
```
$ ask new --template-url https://ffff.com
Please follow the wizard to start your Alexa skill project ->
? Choose a method to host your skill's backend resources:  AWS Lambda
  Host your skill code on AWS Lambda (requires AWS account).
[Warn]: CLI is about to download the skill template from unofficial template https://ffff.com. Please make sure you understand the source code to best protect yourself from malicious usage.
? Would you like to continue download the skill template?  Yes
? Please type in your skill name:  ffff
? Please type in your folder name for the skill project (alphanumeric):  ffff
fatal: repository 'https://ffff.com/' not found
[Error]: CliError: Error: Command failed: git clone --branch ask-cli-x https://ffff.com "/Users/nongwang/Amazon/Github/ask-cli/ffff"  --quiet
```

Do you think we need to fail ahead before asking for any further details since we decide to loose the validation? Another mindset would be change the validation to be `isGitCloneableUrl`?

Reply:
I think the current error message is clear. How would we know if the url is git clonable unless we try cloning it?

Reply:
`ls-remote` might work, but is not guaranteed same as the git-clone's behavior. We will stay tuned for customer's feedback over this experience.